packages/core/src/builder.class.ts

```diff
@@ -2525,15 +2538,15 @@ export class Builder {
       };
     }
 
-    const fn = format === 'solid' || format === 'react' ? 'codegen' : 'query';
+    const fn = format === 'solid' || format === 'react' ? 'codegen' : 'content';
+    let url =
+      fn === 'codegen'
+        ? `${host}/api/v3/${fn}/${this.apiKey}/${keyNames}`
+        : `https://cdn.builder.io/api/v3/${fn}/${queue[0].model}`;
```

Comment:
lets reuse `host` for consistency
```suggestion
        : `${host}/api/v3/${fn}/${queue[0].model}`;
```